pcdsutils/ophyd_helpers.py

```diff
@@ -0,0 +1,242 @@
+"""
+ophyd Device/Signal-related helpers.
+"""
+
+import asyncio
+import contextlib
+import logging
+import time
+from typing import Callable, List, Optional
+
+import ophyd
```

Comment:
New dependency? Do we add this to the meta.yaml/requirements.txt or do we take it as optional? If optional, is it ok to leave it exposed and assume nobody imports here if they don't have ophyd?

Previously we had an implied ophyd dependency in the log submodule, but it was not tied to an import.